runtime/dolphin/src/lib.rs

```diff
@@ -241,10 +246,13 @@ impl Contains<Call> for BaseFilter {
 				| manta_collator_selection::Call::remove_collator{..}
 				| manta_collator_selection::Call::leave_intent{..})
 			| Call::Balances(_)
+			| Call::XTokens(_)
 			| Call::Utility(_) => true,
+			// Filter XCM pallets, we only allow transfer with XTokens.
```

Comment:
You also need to add `AssetManager` here.

Reply:
I left it unfiltered on purpose.
`AssetManager`'s `T::ModifierOrigin` is configured to Root so all of the extrinsics bypass this filter right now.
But if we ever add public user callable extrinsic, or change the `ModifierOrigin`, I want to be forced to manually unfilter it. That seems like the safer way to go.

Reply:
oh, cool!